.cloudbuild/ci/doc-tests.yaml

```diff
@@ -3,6 +3,9 @@ steps:
     id: docs-test
     env:
       - WITH_EXTERNAL_LINKS=true
+    entrypoint: /bin/bash
     dir: /src
-    args: ['yarn', 'remark', '/workspace/docs/pages/**/*.mdx', '--frail']
+    args: 
+      - -c 
+      - ln -s /workspace /src/content && yarn markdown-lint-external-links
```

Comment:
Commenting here because I got added as a reviewer and I don't see a link to the original PR.

I don't love that this creates behavior that differs locally versus when run on GCB. While I'm fine with this as a temporary fix, it would be nice to create a follow up issue to investigate a better fix.

Reply:
We're at the mercy of to conflicting restrictions here: 

1. We can't  use `make -C build.assets test-docs` on a buildbox image as you would locally, as this needs Docker-in-Docker for CI, which we don't want to allow, and
2. There is also no way we can customize the mount point that GCB uses (it is _always_ `/workspace`), so we can't change that to move the docs source into place.

The actual `test-docs` task looks like this:

```make
DOCSBOX=quay.io/gravitational/next:main

test-docs: docsbox
	docker run --platform=linux/amd64 -i $(NOROOT) -v $$(pwd)/..:/src/content $(DOCSBOX) \
		/bin/sh -c "yarn markdown-lint-external-links"
```

...so the build steps in the CI build file are the closest equivalent I could make that would work under GCB